geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeaveJUnitTest.java

```diff
@@ -188,6 +190,26 @@ public boolean isCoordinator() {
 
   }
 
+  @Test
+  public void testFindCoordinatorPausesWhenLocatorWaitTimeIsSet() throws Exception {
```

Comment:
I think that what this is really testing is `findCoordinatorThrowsWhenInterruptedWhileTryingToReconnect`. Alas, testing that something actually waits is hard.

Should we verify that `tcpClientWrapper.sendCoordinatorFindRequest` actually gets called?